cloud: add contents about Connecting to TiDB Dedicated via Private Service Connect on Google Cloud (#14450)
pingcap/docs: tidb-cloud/set-up-private-endpoint-connections.md

---

> title: Connect to TiDB Dedicated via Private Endpoint with AWS

Author: ```suggestion
title: Connect to a TiDB Dedicated Cluster via Private Endpoint with AWS
```

summary: Learn how to connect to your TiDB Cloud cluster via private endpoint with AWS.
---

> # Connect to TiDB Dedicated via Private Endpoint with AWS

Author: ```suggestion
# Connect to a TiDB Dedicated Cluster via Private Endpoint with AWS
```

> ## Restrictions

Author: Google Cloud 中的 restrictoins 是否也适用于 AWS？https://github.com/pingcap/docs/pull/14450/files#diff-34ceaba68d9c7c2b620d2737b72411ea5839cc6b20701345730c5a05d8638c1dR28 @Frank945946

Reviewer: 只有部分适用，就是 endpoint 和 Cluster 需要在同 region，以及单个 Cluster  的 endpoint 上限是 12 个

Author: ok 加上了

> - Private endpoint connection across regions is not supported.

Author: ```suggestion
- You can create up to 12 endpoints for each TiDB Dedicated cluster.
- The private endpoint and the TiDB cluster must be located in the same region.
```

## Restrictions

> - You can create up to 12 endpoints for each TiDB Dedicated cluster.

Author: ```suggestion
- Only the organization owner and the project owner can create Google Cloud Private Service Connect endpoints.
- You can create up to 12 endpoints for each TiDB Dedicated cluster.
```

> - Only the organization owner and the project owner can create Google Cloud Private Service Connect endpoints.

Reviewer 2: ```suggestion
- Only the `Organization Owner` and the `Project Owner` roles can create private endpoints.
```

> - The private endpoint and the TiDB cluster must be located in the same region.

Reviewer 2: ```suggestion
- The private endpoint and the TiDB cluster to be connected must be located in the same region.
```

## Restrictions

- Only the organization owner and the project owner can create Google Cloud Private Service Connect endpoints.

> - You can create up to 12 endpoints for each TiDB Dedicated cluster.

Author: ```suggestion
- You can create up to 10 endpoints for each TiDB Dedicated cluster.
```

1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click the name of your target cluster to go to its overview page.

> 2. Click **Connect** in the upper-right corner. A connection dialog is displayed.

Reviewer: 1和2 合并成一步，点击集群列表页。。。 进入 Connect 页面即可

> 1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click the name of your target cluster to go to its overview page.
> 2. Click **Connect** in the upper-right corner. A connection dialog is displayed.

Author: ```suggestion
1. On the [**Clusters**](https://tidbcloud.com/console/clusters) page, click **...** in the **Action** column.
2. Click **Connect**. A connection dialog is displayed.
```

> 3. Select the **Private Endpoint** tab, and then select the **Google Cloud** tab. The private endpoint you just created is displayed.

Author: ```suggestion
3. Select the **Private Endpoint** tab, and then select the **AWS** tab. The private endpoint you just created is displayed.
```

> 3. Select the **Private Endpoint** tab, and then select the **AWS** tab. The private endpoint you just created is displayed under **Step 1: Create Private Endpoint**.

Reviewer: ```suggestion
3. Select the **Private Endpoint** tab. The private endpoint you just created is displayed under **Step 1: Create Private Endpoint**.
```

> 3. Select the **Private Endpoint** tab, and then select the **AWS** tab. The private endpoint you just created is displayed.
> 4. In the **Action** column, click **Connect**, click the tab of your preferred connection method, and then connect to your cluster with the connection string. The placeholders `<cluster_endpoint_name>:<port>` in the connection string are automatically replaced with the real values.

Author: ```suggestion
3. Select the **Private Endpoint** tab, and then select the **AWS** tab. The private endpoint you just created is displayed under **Step 1: Create Private Endpoint**.
4. Under **Step 2: Connect your application**, click **Connect**, click the tab of your preferred connection method, and then connect to your cluster with the connection string. The placeholders `<cluster_endpoint_name>:<port>` in the connection string are automatically replaced with the real values.
```

> 4. Under **Step 2: Connect your application**, click **Connect**, click the tab of your preferred connection method, and then connect to your cluster with the connection string. The placeholders `<cluster_endpoint_name>:<port>` in the connection string are automatically replaced with the real values.

Reviewer: ```suggestion
4. Under **Step 2: Connect your connection**, click **Connect**, click the tab of your preferred connection method, and then connect to your cluster with the connection string. The placeholders `<cluster_endpoint_name>:<port>` in the connection string are automatically replaced with the real values.
```
